Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Cursors/BaseCursor.cs

```diff
@@ -116,7 +116,8 @@ public float DefaultCursorDistance
         /// <inheritdoc />
         public virtual void SetVisibility(bool visible)
         {
-            if (PrimaryCursorVisual != null)
+            if (PrimaryCursorVisual != null &&
+                PrimaryCursorVisual.gameObject.activeInHierarchy)
```

Comment:
This is very confusing. We check to see if the object is active before we potentially set it active?

Reply:
Yeah this is what broke things haha.